examples/plot_derivative.py

```diff
@@ -0,0 +1,79 @@
+"""
+Derivatives
+===========
+This example demonstrates how to use a set of derivative operators, namely
```

Comment:
`a set of derivative operators` -> `pylops-mpi's derivative operators`

Reply:
Done..